src/views/base_view.js

```diff
@@ -10,6 +12,7 @@ export default AmpersandView.extend({
     AmpersandView.prototype.render.apply(this, arguments);
 
     this.afterRender();
+    setTimeout(this.adjustHeight, 0);
```

Comment:
By putting the adjustHeight call in the base view, you're going to send a resize event for every single view in the entire app. This is going to lead to some funny jank. This should be called once, in the index view's afterRender method, right after the subviews are all rendered.